blocks/columns/columns.css

```diff
@@ -31,3 +31,155 @@
     order: unset;
   }
 }
+
+.columns-feature-card {
+  & > .columns-feature-card-list {
+    padding: 0;
+    margin: 0;
+
+    & .columns-feature-card-item {
+      list-style: none;
+
+      > p:first-child {
+        font-family: var(--heading-font-family);
+        color: var(--color-white);
+        font-size: 28px;
+        line-height: 31px;
+        margin-top: 0;
+        margin-bottom: 24px;
+        text-transform: uppercase;
+      }
+
+      .columns-feature-card-description {
+        font-family: var(--heading-font-family);
+        font-size: 18px;
+        font-weight: 400;
+        line-height: 25px;
+        text-align: left;
+        color: var(--color-white);
```

Comment:
all colors, size, weight, line-height are set at template level in context of background color.

Reply:
they don't match with actual sizes within. the designs. There is some inconsistency. Also, these sizes change for different viewport. That's the reason for override. We can come back after discussion with customer.